src/WebAPI/Controllers/TicketsController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Models;
+
+namespace WebAPI.Controllers
+{
+    [Produces("application/json")]
```

Comment:
Have you tried to use API without this attribute? I think it should work by default as 'application/json'